app/mailers/application_mailer.rb

```diff
@@ -1,4 +0,0 @@
-class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
-  layout 'mailer'
-end
```

Comment:
Removing mailer when you use Devise might not be the best thing. Not a blocker but be prepared to add this back in when making use of some of the devise functionality (like sending emails to new users for confirmation, etc)

Reply:
will keep this in mind for the future. We will try to make more a conscious effort to stay in scope 👍